empyrion/egg-empyrion--galactic-survival.json

```
@@ -1,22 +1,22 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
     "meta": {
-        "version": "PTDL_v2",
+        "version": "PLCN_v1",
         "update_url": null
     },
-    "exported_at": "2024-06-01T00:04:10+00:00",
+    "exported_at": "2025-02-25T15:21:08+00:00",
     "name": "Empyrion: Galactic Survival",
     "author": "eggs@goover.dev",
-    "uuid": "3a1450cf-20a8-4fa9-98d1-f38c9f1b5636",
+    "uuid": "fe2bcb9e-3e46-41c7-80a7-d8fbfb98a7f0",
```

```suggestion
    "uuid": "3a1450cf-20a8-4fa9-98d1-f38c9f1b5636",
```
UUID shouldn't change.